src/assets/wise5/classroomMonitor/classroomMonitorComponents/nodeGrading/nodeGradingView/nodeGradingView.ts

```diff
@@ -359,6 +364,14 @@ export class NodeGradingViewController {
   showReport($event) {
     this.MilestoneService.showMilestoneDetails(this.milestoneReport, $event);
   }
+
+  showPeerGroupGroupings(nodeId: string, componentId: string): void {
+    this.NodeService.showPeerGroupDetails(
+      this.TeacherDataService.getCurrentPeriod().periodId,
+      nodeId,
+      componentId
+    );
+  }
```

Comment:
What's the reason for putting showPeerGroupDetails() in NodeService when all it does is open a dialog? Can that be done in this function instead?

Reply:
nodeGradingView.ts is old Angular. PeerGroupDialogComponent.ts is a new Angular component. Right now it's easier to open the dialog from NodeService.ts since NodeService is a new Angular service. Maybe we could downgrade PeerGroupDialogComponent and try to open it from the old Angular world if we really wanted to. Is it worth it?